examples/direct_fidelity_estimation.py

```diff
@@ -173,19 +175,19 @@ class PauliTrace:
 
 
 def _estimate_pauli_traces_clifford(
-    n_qubits: int, clifford_state: cirq.CliffordState, n_measured_operators: Optional[int]
+    n_qubits: int, clifford_tableau: cirq.CliffordTableau, n_measured_operators: Optional[int]
 ) -> List[PauliTrace]:
     """
     Estimates the Pauli traces in case the circuit is Clifford. When we have a
     Clifford circuit, there are 2**n Pauli traces that have probability 1/2**n
     and all the other traces have probability 0. In addition, there is a fast
     way to compute find out what the traces are. See the documentation of
-    cirq.CliffordState for more detail. This function uses the speedup to sample
+    cirq.CliffordTableau for more detail. This function uses the speedup to sample
     the Pauli states with non-zero probability.
 
     Args:
         n_qubits: An integer that is the number of qubits.
-        clifford_state: The basis of the Pauli states with non-zero probability.
+        clifford_tableau: The basis of the Pauli states with non-zero probability.
```

Comment:
This description does not seem accurate for CliffordTableau

Reply:
Indeed, but maybe with the change you requested above, the comment is actually accurate now (the variable is stabilizer_basis).

Reply:
Looks good 👍

Reply:
Ack.